tests/shared/src/test/scala/cats/tests/XorTests.scala

```diff
@@ -12,6 +13,8 @@ import org.scalacheck.Arbitrary._
 import scala.util.Try
 
 class XorTests extends CatsSuite {
+  checkAll("Xor[String, Int]", algebra.laws.GroupLaws[Xor[String, Int]].monoid)
```

Comment:
Oh you're right, `algebra` includes `Serializable` checks in their laws - @non confirm?